package.json

```diff
@@ -119,6 +119,7 @@
     "react-addons-test-utils": "^15.5.1",
     "react-docgen-typescript-loader": "^2.1.0",
     "react-dom": "^16.2.0",
+    "react-powerplug": "^1.0.0-rc.1",
```

Comment:
It's such a shame that `package.json` can't handle comments like a `Podfile` or `Gemfile`, but thank you for clarifying why this was added 👍

Reply:
@ds300 - `react-powerplug` is super nice with many more uses than just Storybooks. Once some patterns shake out it might be worthwhile to announce to wider group so that others are aware.